freshmaker/logger.py

```diff
@@ -139,24 +127,35 @@ def init_logging(conf):
     """
     Initializes logging according to configuration file.
     """
-    log_format = '%(levelname)s - %(message)s'
+    log = logging.getLogger()
+    log.setLevel(conf.log_level)
+
+    formatter = logging.Formatter(
+        "[%(asctime)s.%(msecs)d] [%(processName)s - %(threadName)s] [%(levelname)s] %(message)s",
+        "%Y-%m-%dT%H:%M:%S",
+    )
+    console_handler = logging.StreamHandler(stream=sys.stdout)
+    console_handler.setLevel(conf.log_level)
+    console_handler.setFormatter(formatter)
+    log.addHandler(console_handler)
+
     log_backend = conf.log_backend
 
     if not log_backend or len(log_backend) == 0 or log_backend == "console":
-        logging.basicConfig(level=conf.log_level, format=log_format)
-        log = logging.getLogger()
-        log.setLevel(conf.log_level)
+        pass
+
     elif log_backend == "journal":
-        logging.basicConfig(level=conf.log_level, format=log_format)
         try:
             from systemd import journal
         except ImportError:
             raise ValueError("systemd.journal module is not installed")
-
-        log = logging.getLogger()
-        log.propagate = False
         log.addHandler(journal.JournalHandler())
+
+    elif log_backend == "file":
+        file_handler = logging.FileHandler(conf.log_file)
+        file_handler.setFormatter(formatter)
+        log.removeHandler(console_handler)
```

Comment:
why not moving `log.addHandler(console_handler)` to the "console" and "journal" branches?

I don't see any requirements about writing the logs to journal log, probably we can remove it in future improvements.